src/android/CHIPTool/app/src/main/res/layout/callback_item.xml

```diff
@@ -0,0 +1,44 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/callbackRow"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal">
```

Comment:
Can remove the lines `xmlns:tools=...` and `android:orientation=...`, these are unused.